src/dotnet-scaffolding/dotnet-scaffold/Telemetry/CommandExecuteTelemetryEvent.cs

```diff
@@ -8,14 +9,19 @@ namespace Microsoft.DotNet.Tools.Scaffold.Telemetry;
 internal class CommandExecuteTelemetryEvent : TelemetryEventBase
 {
     private const string TelemetryEventName = "CommandExecute";
-    public CommandExecuteTelemetryEvent(DotNetToolInfo dotnetToolInfo, CommandInfo commandInfo, int? exitCode) : base(TelemetryEventName)
+    public CommandExecuteTelemetryEvent(DotNetToolInfo dotnetToolInfo, CommandInfo commandInfo, int? exitCode, string? chosenCategory = null) : base(TelemetryEventName)
     {
         SetProperty("PackageName", dotnetToolInfo.PackageName);
         SetProperty("Version", dotnetToolInfo.Version);
         SetProperty("ToolName", dotnetToolInfo.Command);
         SetProperty("ToolLevel", dotnetToolInfo.IsGlobalTool ? TelemetryConstants.GlobalTool : TelemetryConstants.LocalTool);
         SetProperty("CommandName", commandInfo.Name);
-        SetProperty("CommandCategory", commandInfo.DisplayCategory);
+        SetProperty("AllCommandCategories", string.Join(",", commandInfo.DisplayCategories));
+        if (!string.IsNullOrEmpty(chosenCategory))
+        {
+            SetProperty("ChosenCategory", chosenCategory);
```

Comment:
Just noting generally that from our discussion earlier we'll want to go back and make these two new properties hashed.

Reply:
Is [this](https://github.com/dotnet/Scaffolding/blob/b4afd5d143d15dfd6dbb1a8efd0465463e9b1d5c/src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Internal/Extensions/StringExtensions.cs#L152) enough for hashing?

Reply:
Yes, that should be good as far as I'm aware